examples/linear_model/plot_logistic_multinomial.py

```diff
@@ -0,0 +1,68 @@
+"""
+====================================================
+Plot multinomial and One-vs-Rest Logistic Regression
+====================================================
+
+Plot decision surface of multinomial and One-vs-Rest Logistic Regression.
+The hyperplanes corresponding to the three One-vs-Rest (OVR) classifiers
+are represented by the dashed lines.
```

Comment:
Maybe, you should combine this with the already existing `plot_iris_logistic.py`